policyengine_us/variables/gov/states/oh/tax/income/deductions/oh_unreimbursed_medical_care_expense_deduction.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_us.model_api import *
+
+
+class oh_unreimbursed_medical_care_expense_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Ohio Unreimbursed Medical and Health Care Expense Deduction"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://tax.ohio.gov/static/forms/ohio_individual/individual/2022/it1040-sd100-instruction-booklet.pdf#page=18",  # Line 36
+        "https://tax.ohio.gov/static/forms/ohio_individual/individual/2022/it1040-sd100-instruction-booklet.pdf#page=27",
+        "https://codes.ohio.gov/ohio-revised-code/section-5747.01",  # R.C. 5747.01(10)
+    )
+    defined_for = StateCode.OH
+
+    def formula(tax_unit, period, parameters):
+        premiums_expenses = add(
+            tax_unit, period, ["health_insurance_premiums"]
```

Comment:
this is only premiums that aren't paid in part by employers. we'll need some data work to do this, see HIPAID

https://www2.census.gov/programs-surveys/cps/datasets/2021/march/asec2021_ddl_pub_full.pdf

I'll come back to this with more specific guidance

Reply:
@MaxGhenis friendly reminder

Reply:
Let's add a variable mapping to `HIPAID` from the ASEC.

name: `employer_premium_contribution`
type: Enum
values: `NONE`, `SOME`, `ALL`, `N/A`